packages/gatsby-plugin-mdx/gatsby/on-create-node.js

```diff
@@ -6,7 +6,6 @@ const { createContentDigest } = require(`gatsby-core-utils`)
 const defaultOptions = require(`../utils/default-options`)
 const createMDXNode = require(`../utils/create-mdx-node`)
 const { MDX_SCOPES_LOCATION } = require(`../constants`)
-const { findImports } = require(`../utils/gen-mdx`)
 
 const contentDigest = val => createContentDigest(val)
```

Comment:
this can also removed? and the exisiting `contentDigest` replaced by `createContentDigest`, maybe also not needed as param at `cacheScope`

Reply:
![image](https://user-images.githubusercontent.com/209817/87536411-57968680-c699-11ea-9c07-e1e415ab6c4c.png)